Stack/Opc.Ua.Core/Stack/Generated/Opc.Ua.PredefinedNodes.xml

```diff
@@ -1928,7 +1928,7 @@
       <uax:NumericId>15959</uax:NumericId>
       <uax:Value>
         <uax:Value>
-          <uax:String>1.04.10</uax:String>
+          <uax:String>1.04.11-RC</uax:String>
```

Comment:
was the version change intentional?

Reply:
Yes. The NodeSet is not byte for byte identical to the original which means we need a version update. I added the errata and put it on the agenda for next weeks call.